test/run_tests.py

```diff
@@ -46,8 +46,9 @@ def acceptance_tests(interpreter, browser, args):
 
 def start_http_server():
     server_output = TemporaryFile()
+    #server_output = open("server.log", 'w+', buffering=False)
```

Comment:
No, we should not remove the comment. We should add a comment describing that by default the output goes to a temporary file, but toggling the commented lines allows developer to keep the output.